src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -108,7 +108,7 @@
   <Target Name="PreCloudBuild">
     <!-- copy runner scripts so they can be uploaded as supplemental payload -->
     <ItemGroup>
-      <RunnerScripts Include="$(ToolsDir)RunnerScripts/**/*.py" />
+      <RunnerScripts Include="$(ToolsDir)RunnerScripts/**/*.*" />
```

Comment:
*.* means that whatever is in these directories, even if it's not checked into git, will end up deployed. That means deploying from _your_ machine will produce different results than deploying from _my_ machine, which is bad.  Each individual file should be included in here, so that the build/deployment is predictable and repeatable.

And each of the 4 files (and realistically, I'd like to see the *.py exploded as well, but it's far less likely a rogue .py file will be here than, say, some random *.exe that is auto-ignored by git)